src/node.ts

```diff
@@ -1,5 +1,8 @@
 import getLDRequestHandler from "./requestHandler"
-export { getLDRequestHandler }
+
+export default {
+  getLDRequestHandler,
+}
```

Comment:
does that mean a major change?

Reply:
Isn't that still the same for importing when exporting it in a envelope? Like you can destructure it...

Reply:
oh my I think that snuck in accidentally, let me check!